src/lib/storage/abstract_device_factory.rb

```diff
@@ -300,7 +296,6 @@ def check_param(name, param)
       # @param child  [String] name of child  factory product
       #
       def check_hierarchy(parent, child)
-        # rubocop:disable Style/GuardClause
```

Comment:
it is actually auto correction from rubocop, probably only one half is implemented